lib/resultuploaders/result_uploader.rb

```diff
@@ -10,24 +11,48 @@ class ResultUploader
     # UploaderFactory
     #
     class UploaderFactory
-      UPLOADERS = {
-        dropbox: Dropbox
-      }.freeze
+      def self.initialize(logger)
```

Comment:
Why did you put this in a class method? can't you just run the code in the body of UploaderFactory?

Reply:
Oh, I missed the logger usage. In my opinion:
- Let `try_load_uploader` fail when a class is not found. Such a case is almost certainly a bug.
- Let the default exception handler handle load errors. It should tell the context enough to debug in case of failure.

Reply:
You are right. The default exception handler output is good for debugging.